### PR TITLE
Named branches

### PR DESCRIPTION
**Pull request by maxsolomon14**

### CODE REVIEW

File: app/Http/Controllers/MessagesController.php [hunk 1]
```diff
@@ -29,7 +29,9 @@ public function index()
             return $message['sender_id'] + $message['recipient_id'];
         });
 
-        return view('pages.messages')->withMessages($messages);
+        $findUser = User::all();
```
Comment (alexroden) [inline app/Http/Controllers/MessagesController.php]: Could probably put this into a composer file

File: resources/views/pages/profile.blade.php [hunk 2]
```diff
@@ -11,25 +11,24 @@
...
         @endforeach
-
-        <h3>Here are all of {{$user->name}}'s comments:</h3>
+            <h3>Here are all of {{$user->name}}'s comments:</h3>
 
         @foreach ($user->comments as $comment)
         <ul class="list-group">
             <li class="list-group-item">
-                <small>Written by <a href="/profile/{{$comment->user_id}}">{{$comment->user_name}}</a> on {{$comment->created_at->format('d F Y')}}  -  Go to <a href="../post/{{$comment->post_id}}">post?</a></small><br>   
+                <small>Written by <a href="{{route('profile', $comment->user_id)}}">{{$comment->user_name}}</a> on {{$comment->created_at->format('d F Y')}}  -  Go to <a href="{{route('post.show', $comment->post_id)}}">post?</a></small><br>
```
Comment (alexroden) [inline resources/views/pages/profile.blade.php]: Wait.... have you put a user relationship on the comments and also included the user's name? Why?

Comment (maxsolomon14) [reply resources/views/pages/profile.blade.php]: @alexroden I think I have a user's name in the comments table. Is that what you mean?

File: app/Http/Controllers/MessagesController.php (hunk 1 above)
Comment (maxsolomon14) [reply app/Http/Controllers/MessagesController.php]: @alexroden very true, I'll get on it

File: resources/views/pages/profile.blade.php (hunk 2 above)
Comment (alexroden) [reply resources/views/pages/profile.blade.php]: Yeah, why would you need the user's name in the comments table, if you have the user relation? Just use that

Comment (maxsolomon14) [reply resources/views/pages/profile.blade.php]: You're right, I'll get on it. I've also added the $findUser in the composer

Comment (maxsolomon14) [reply resources/views/pages/profile.blade.php]: Also why have I named this branch 'named branches' when it's meant to be 'named routes' ignore that!

**Pull request merged**